README.md

```diff
@@ -11,6 +11,8 @@ m-LoRA (a.k.a Multi-Lora Fine-Tune) is an open-source framework for fine-tuning
 
 - Multiple LoRA Adapters: Support for concurrent fine-tuning of multiple LoRA/QLoRA adapters.
 
+- LoRA based Mix-of-Expert: Support for MixLoRA, which implements Mix-of-Expert architecture based on multiple LoRA adapters for frozen FFN layer.
```

Comment:
can you provide the related doc to explain this to here?

Reply:
I will add a detailed documentation after this pr.